input/full.go

```diff
@@ -19,6 +19,19 @@ func CollectFull(server *state.Server, connection *sql.DB, globalCollectionOpts
 
 	ps.CollectedAt = time.Now()
 
+	if server.Config.SkipIfReplica {
+		var isReplica bool
+		isReplica, err = postgres.GetIsReplica(logger, connection)
+		if err != nil {
+			logger.PrintError("Error checking replication status")
```

Comment:
I wonder if we could hit permission issues here? (not sure if `pg_is_in_recovery` is accessible to all users)

Also, probably should double check special cases, like here: https://github.com/pganalyze/collector/blob/master/input/postgres/replication.go#L86

Thus, I'm wondering if we shouldn't accept failure here, and continue with collection if we can't check replication status.